composer.json

```diff
@@ -8,7 +8,7 @@
         }
     },
     "require": {
-        "php": "^7.3 || ^8.0"
+        "php": "^7.4 || ^8.0"
```

Comment:
https://github.com/Ocramius/ProxyManager/pull/628 drops support for PHP 7.3 and quite frankly, I don't want to support it either.

Reply:
Yep i agree